lib/plugins/aws/deploy/compile/events/apiGateway/lib/methods.js

```diff
@@ -1,7 +1,10 @@
 'use strict';
 
-const BbPromise = require('bluebird');
 const _ = require('lodash');
+const BbPromise = require('bluebird');
+const pathLib = require('path');
+
+const naming = require(pathLib.join(__dirname, '..', '..', '..', '..', '..', 'lib', 'naming.js'));
```

Comment:
Absolutely, I thought I'd removed all instances of this from the code.  Thanks for noticing it.